src/gsi-ncdiag/test_gsidiag.py.in

```diff
@@ -20,10 +20,22 @@ parser.add_argument('-t', '--type',
 parser.add_argument('-p', '--platform',
                     help="platform to run, only works on conv",
                     type=str)
+parser.add_argument('-b', '--add_obsbias',
+                    help="add obsbias groups, only works on rad",
+                    type=str)
+parser.add_argument('-q', '--add_qcvars',
+                    help="add qc variables, only works on rad",
+                    type=str)
+parser.add_argument('-r', '--add_testrefs',
+                    help="add test variables, only works on rad",
+                    type=str)
```

Comment:
Change  '**-t'**,     '--add_testvars'    to '**-r**',    '--add_testrefs' due to conflict of using '-t '
( '-t' is already used by data type )